AsyncDisplayKit/ASImageNode+AnimatedImage.mm

```diff
@@ -85,6 +85,23 @@ - (void)coverImageCompleted:(UIImage *)coverImage
   }
 }
 
+- (NSString *)animatedImageRunLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+  return _animatedImageRunLoopMode;
+}
+
+- (void)setAnimatedImageRunLoopMode:(NSString *)runLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+
+  if (_displayLink != nil) {
+    [_displayLink removeFromRunLoop:[NSRunLoop mainRunLoop] forMode:_animatedImageRunLoopMode];
+    [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:runLoopMode];
```

Comment:
@ejensen this is a very smart idea! I would be open to changing the default value here as well, but providing the setter method is a good idea. However, could you add a check for at least a nil string?  Presumably in that case we should set it back to the default. I'm not sure if there is a way to validate that a string is a real mode though.

cc @garrettmoon for overall thoughts, as well as ideas on the default.